### PR TITLE
Add a service that returns the list of Channels to which the customer should be added

### PR DESCRIPTION
**Pull request by LucaGallinari**
Fixes #23

### CODE REVIEW

**Comment by LucaGallinari** [comment]
Moreover, we should find a way to test/assert the fact that we enqueue the customer only for the channels returned from the method call to the repository, and not all channels as we are doing now. This should be possible with some mock.

File: src/Repository/ActiveCampaignChannelRepositoryInterface.php [hunk 1]
```diff
@@ -11,6 +11,4 @@
  */
 interface ActiveCampaignChannelRepositoryInterface extends ActiveCampaignResourceRepositoryInterface
 {
-    /** @return ChannelInterface[] */
-    public function findAllEnabledForActiveCampaign(): array;
 }
```
Comment (lruozzi9) [inline src/Repository/ActiveCampaignChannelRepositoryInterface.php:14]: This should be removed. SO it should be changes also the ChannelRepository in tests/Application and changed the the installation documentation.

**Review by lruozzi9 — CHANGES_REQUESTED** [review]
Can you also rebase this PR and fix the documentation on First Setup page?

File: src/Repository/ActiveCampaignChannelRepositoryInterface.php [hunk 2]
```diff
@@ -5,12 +5,13 @@
...
+use Sylius\Component\Core\Model\CustomerInterface;
 
 /**
  * @extends ActiveCampaignResourceRepositoryInterface<ChannelInterface>
  */
 interface ActiveCampaignChannelRepositoryInterface extends ActiveCampaignResourceRepositoryInterface
 {
     /** @return ChannelInterface[] */
-    public function findAllEnabledForActiveCampaign(): array;
+    public function findAllForCustomer(CustomerInterface $customer): array;
```
[the remark this answers is not in the document]
Comment (LucaGallinari) [reply src/Repository/ActiveCampaignChannelRepositoryInterface.php]: Mhmm.. this is an old comment i think? This was the old implementation with the repository

File: src/Repository/ActiveCampaignChannelRepositoryInterface.php (hunk 1 above)
Comment (LucaGallinari) [reply src/Repository/ActiveCampaignChannelRepositoryInterface.php]: Ok, i think i got what you mean. Take a look at the last commit

File: src/Repository/ActiveCampaignChannelRepositoryInterface.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (lruozzi9) [reply src/Repository/ActiveCampaignChannelRepositoryInterface.php]: Yes, sorry. It was in memory 🤦‍♂️

File: src/Repository/ActiveCampaignChannelRepositoryInterface.php (hunk 1 above)
Comment (lruozzi9) [reply src/Repository/ActiveCampaignChannelRepositoryInterface.php]: Perfect!

**Review by lruozzi9 — APPROVED** [review]

**Pull request merged**